ioctl/newcmd/account/accountsign_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/test/mock/mock_ioctlclient"
+)
+
+func TestNewAccountSign(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslationString", config.English).AnyTimes()
+
+	// test for not invalid account address
```

Comment:
could you add a valid test?

Reply:
The test will be blocked to wait for password when invoke `util.ReadSecretFromStdin()`, have any solution?

Reply:
This is the test code
```
// test for exist account
testWallet := filepath.Join(os.TempDir(), "testNewAccountSign")
defer testutil.CleanupPath(t, testWallet)

originWallet := config.ReadConfig.Wallet
config.ReadConfig.Wallet = testWallet
defer func() {
config.ReadConfig.Wallet = originWallet
}()
ks := keystore.NewKeyStore(config.ReadConfig.Wallet, keystore.StandardScryptN, keystore.StandardScryptP)
require.NotNil(ks)

// create accounts
cmd = NewAccountSign(client)
account, err := ks.NewAccount("test")
require.NoError(err)
addr, err := address.FromBytes(account.Address.Bytes())
require.NoError(err)
require.NoError(cmd.Flag("signer").Value.Set(addr.String()))
result, err := util.ExecuteCmd(cmd)
require.NoError(err)
require.Equal("", result)
```

Reply:
> 

`ReadSecret()` in client can be used for this pr. And use mocked one in the test

Reply:
Should I refactor the `ioctl/newcmd/account/account.Sign()`, let the new implement read password by `client.ReadSecret()`?

Reply:
sure

Reply:
Make sense of it.

Reply:
Done, PTAL.